components/proxy/src/test/java/com/hotels/styx/serviceproviders/ServiceFactoriesConfigTest.java

```diff
@@ -95,58 +95,22 @@
             "    class: " + MyFactoryA.class.getName() + "\n" +
             "    config: {configValue: unexpectedValue2}\n";
 
-    @BeforeMethod
-    public void startRecordingLogs() {
-        log = new LoggingTestSupport(ServiceFactoriesConfig.class);
-    }
+//    @BeforeMethod
```

Comment:
The Set-Up and tests are commented out.

Reply:
Oh, that's because ServiceFactoriesConfig class no longer exists. I should have removed this class too.